Documentation/components/iptables/pcn-iptables.rst

```diff
@@ -129,4 +143,20 @@ Limitations
 ^^^^^^^^^^^
 
 - It requires your network interfaces to support XDP Native mode
-- If any interface is not supporting XDP, on such interface traffic is not filtered
\ No newline at end of file
+- If any interface is not supporting XDP, on such interface traffic is not filtered
+
+
+pcn-iptables components
+-----------------------
+
+iptables submodule
+^^^^^^^^^^^^^^^^^^
+
+Under :scm_web:`src/components/iptables/iptables <src/components/iptables>` a customized fork of iptables is included as submodule.
```

Comment:
Turn the sentence the other way around.
A customized fork of iptables is included as submodule under...